app/src/main/java/edu/uc/hornunjb/weatherwear/MainActivity.kt

```diff
@@ -45,10 +40,15 @@ class MainActivity : AppCompatActivity() {
             val city = response.name
             val humidity = response.main.humidity.toString() + "%"
             val pressure = response.main.pressure
-            val tempRange = round((response.main.temp_min -273)*9/5 + 32).toString() + " to " + round((response.main.temp_max -273)*9/5 + 32).toString() + " Degrees"
-            Toast.makeText(this, tempRange, Toast.LENGTH_LONG).show()
```

Comment:
I took this toast out because it is not necessary to the program and it is a redundant piece of data. Duplication is one of sonarQube's 7 deadly sins.